generators/generator-base.js

```diff
@@ -199,6 +304,23 @@ module.exports = class extends PrivateBase {
         this.needleApi.clientI18n.addEntityTranslationKey(key, value, language);
     }
 
+    /**
+     * Add a new menu entry in the "global.json" translations.
+     *
+     * @param {string} key - Key for the entity name
+     * @param {string} value - Default translated value
+     * @param {string} language - The language to which this translation should be added
+     */
+    addMenuTranslationKey(key, value, language, entityModule) {
```

Comment:
Refactor hint:
  - there are 4 parameters (which may be too much)
  - everytime we use the entityModule param to determine what method to call with the other 3
  - the three params might belong to the same object (or structure) and passing them in that order, everytime, isn't that pretty